.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2.0
+
+jobs:
+  build:
+    docker:
+      - image: cibuilds/docker:19.03
+    steps:
+      - checkout
+      - setup_docker_php
```

Comment:
What does this do? I guess we don't need `php` just docker, which I guess we got already in the image you have used above.

Reply:
Yeah, actually I forgot to remove the setup step, my bad